frontend/src/components/SharedLayout/index.jsx

```diff
@@ -1,8 +1,8 @@
 import { Outlet } from "react-router-dom";
 import { Suspense } from "react";
-import Loader from "components/Loader/Loader";
+import Loader from "components/Loader/index";
```

Comment:
jak pisze się index.jsx to nie trzeba pisać index
import Loader from "components/Loader";
tak bedzie działało